types/auth0/index.d.ts

```diff
@@ -1521,16 +1544,32 @@ export interface DeviceCredential {
     last_used?: string;
 }
 
+export interface sendEnrollmentTicketData {
```

Comment:
```suggestion
export interface SendEnrollmentTicketData {
```
IMO this is not widely used notation for JavaScript/TS interfaces